### ISSUE TITLE
good evening. I founded some errors

### ISSUE THREAD

**Issue by forgotmeplease** [verata-veritatis/bybit-market-maker#8]
2021-02-05 02:39:22 - pybit - INFO - Initializing HTTP session.
2021-02-05 02:39:22 - pybit - INFO - Initializing Authenticated WebSocket.
2021-02-05 02:39:24 - pybit - INFO - Authorization successful.
2021-02-05 02:39:24 - pybit - INFO - Subscription to ['instrument_info.100ms.BTCUSD', 'position', 'order'] successful.
Traceback (most recent call last):
  File "run.py", line 3, in <module>
    market_maker.Application()
  File "/home/qs/Desktop/BYBIT MAKER/market_maker/market_maker.py", line 227, in __init__
    Algorithm().run()
  File "/home/qs/Desktop/BYBIT MAKER/market_maker/market_maker.py", line 174, in run
    median, interval, qty = self.submit_initial()
  File "/home/qs/Desktop/BYBIT MAKER/market_maker/market_maker.py", line 118, in submit_initial
    balance = self.req.get_wallet_balance() * last_price * MARGIN
  File "/home/qs/Desktop/BYBIT MAKER/market_maker/market_maker.py", line 73, in get_wallet_balance
    r = self.session.get_wallet_balance('BTC')
TypeError: get_wallet_balance() takes 1 positional argument but 2 were given

**Comment by verata-veritatis** [comment]
Hey there. Stay tuned, new market maker this weekend. I haven't made any updates for months and pybit has changed a LOT.

**Comment by forgotmeplease** [comment]
yeah, i developed 3 projects for this exchange but they doesn't work now

**Comment by forgotmeplease** [comment]
If you need some help. I will help you.

**Comment by babyrig** [comment]
I am waiting too :)

**Comment by babyrig** [comment]
Hello sir, any update or any idea when will be ready ?

**Comment by bingining** [comment]
Cannot run. I am waiting as well.
